packages/python-packages/doc-warden/warden/index_packages.py

```diff
@@ -3,7 +3,7 @@
 
 from __future__ import print_function
 
-from .warden_common import check_match, walk_directory_for_pattern, get_omitted_files, get_java_package_roots, get_net_packages, get_python_package_roots, get_js_package_roots, find_alongside_file, find_below_file, parse_pom, parse_csproj, is_java_pom_package_pom, find_above_file
```

Comment:
@scbedd correct me if I'm wrong but I believe we can remove this package-indexing code from doc-warden now that we no longer use it to produce the package indexes. We are now doing that in the main azure-sdk repo instead. So I'd prefer we just remove this code all together instead of adding more logic to it.

Reply:
Fine by me ... once Scott confirms I'll perform a package-indexing-ectomy.

Reply:
I've done a bit of a digging exercise on ripping out ```index_packages.py```. Turns out that it is used by ```doc-warden``` when it is verifying the casing of change log files. It needs to discover the package locations based on whatever heuristics are baked into the logic to get package roots, and then it uses the ```PackageInfo``` metadata to look at capitalization issues.

Reply:
That is fair I know we are going to be moving some of this stuff out of doc-warden as well.  Could you instead file an issue to remove the package handling from doc-warden and we can consider cleaning it up later?

FYI @chidozieononiwu @praveenkuttappan

Reply:
The package indexing was an easy way to get important information about all packages in the repo including `package file location` `Changelog and readme location`  and `package version`. Its only used for verifying changelogs as well as Readme and Changelog file case. None of those features are enabled yet.

Would moving out the code mean that DocWarden will now be used for only readmes going forward?

Reply:
Added issue #347 to review the scope of the ```doc-warden``` tool.